applications/ContactStructuralMechanicsApplication/python_scripts/contact_structural_mechanics_explicit_dynamic_solver.py

```diff
@@ -41,6 +41,14 @@ def __init__(self, model, custom_settings):
 
         # Setting default configurations true by default
         auxiliar_methods_solvers.AuxiliarSetSettings(self.settings, self.contact_settings)
+        if not self.dynamic_settings["damping_settings"]["determine_rayleigh_damping"].GetBool():
```

Comment:
as we discussed this will go to a process, so you should update and merge the other PR first

Reply:
Yes, in fact this right now will provoke an error, I am updating this